Gems/PhysX/Code/Source/JointComponent.h

```diff
@@ -73,8 +79,8 @@ namespace PhysX
         };
 
         // AZ::Component
-        void Activate() override;
-        void Deactivate() override;
+        virtual void Activate() override;
+        virtual void Deactivate() override;
```

Comment:
nit: `virtual` keyword can be removed here as `override` is present

Reply:
Done